### PR TITLE
Update IceCap RealmOS and make tests more reliable

### PR DESCRIPTION
**Pull request by mathias-arm**
- Include latest IceCap RealmOS updates
- Work-around for runtime-manager/console-server race-condition (#429)
- Fix for `pkill` status code in `quickstart` GitHub Actions job

### CODE REVIEW

File: scripts/icecap-git-reset.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```
Comment (egrimley-arm) [inline scripts/icecap-git-reset.sh:2]: Could we have a comment explaining what this script is for?

**Review by egrimley-arm — COMMENTED** [review]
You might as well update `icecap/icecap` to `85a5c6900c4ce162bca8bee86f9a0e2af47e65ab` (including `strncmp`) while you're at it.

**Review by gbryant-arm — DISMISSED** [review]
LGTM

File: scripts/icecap-git-reset.sh (hunk 1 above)
Comment (dominic-mulligan-arm) [reply scripts/icecap-git-reset.sh:2]: (It also needs the standard copyright and licensing header at the top of the file.)

File: workspaces/icecap-runtime/src/virtio-console-server/src/main.rs [hunk 2]
```diff
@@ -129,6 +129,11 @@ pub unsafe extern "C" fn virtio_virt_to_phys(vaddr: usize) -> usize {
 
 // entry point
 fn main(config: Config) -> Fallible<()> {
+    // FIXME: Yield 10 times to let the runtime start before the console server
```
Comment (dominic-mulligan-arm) [inline workspaces/icecap-runtime/src/virtio-console-server/src/main.rs:132]: Is this FIXME reflected in the issue tracker so it's not forgotten?

**Review by dominic-mulligan-arm — CHANGES_REQUESTED** [review]

**Comment by mathias-arm** [comment]
#443 tracks a better fix for the for runtime-manager/console-server race-condition

**Review by egrimley-arm — APPROVED** [review]

**Pull request merged**